docs/querying-data.rst

```diff
@@ -0,0 +1,207 @@
+
+The following query calculates the total value of tenders diaggregated by currency and tender status in collection ``1060``.
+
+.. code-block:: sql
+
+  select
+    tender_value_currency, --return the currency of the tender value, values in OCDS have an amount and a currency, as datasets may contain values in multiple currencies
+    tender_status,
+    sum(tender_value_amount)
+  from
+    tender_summary
+  where
+    collection_id = 1060
+  group by
+    tender_value_currency,
+    tender_status
+  order by
+    tender_value_currency,
+    tender_status;
+
+To learn more about the summaries and aggregates in the ``tender_summary`` table, refer to the :doc:`database`.
+
+.. tip::
+
+  The ``tender``, ``award`` and ``contract`` objects in OCDS all have a ``.status`` property.
+
+  Kingfisher Views exposes these status properties in ``tender_summary.tender_status``, ``awards_summary.award_status`` and ``contracts_summary.contract_status``.
+
+  Consider which statuses you want to include or exclude from your analysis, for example you might wish to exclude pending and cancelled contracts when calculating the total value of contracts for each buyer.
+
+  The `OCDS codelist documentation <https://standard.open-contracting.org/latest/en/schema/codelists/#>`__ describes the meaning of the statuses for each object.
+
+Calculate the top 10 buyers by award value
+------------------------------------------
+
+Summary data on buyers is stored in the ``buyer_summary`` table and summary data on awards is stored in the ``award_summary`` table.
+
+To join summary tables, use the ``id`` column, which uniquely identifies a release. To learn more about the relationships between tables refer to the :doc:`database`.
+
+The ``buyer_summary`` table doesn't include the buyer's name, however the ``buyer`` column contains a jsonb blob of the buyer  for each contracting process, which this can be extracted from.
+
+Most summary tables in Kingfisher Views include a column containing jsonb blobs of the object the summary relates to.
+
+The following query calculates the top 10 buyers by award value for collection ``1060``, disaggregated by currency and counting 'active' awards only:
+
+.. code-block:: sql
+
+  select
+      buyer_identifier,
+      buyer -> 'name' as buyer_name, --extract the buyer name from the JSON
+      award_value_currency,
+      sum(award_value_amount) as award_amount
+  from
+      awards_summary
+  join
+      buyer_summary on awards_summary.id = buyer_summary.id
+  where
+      awards_summary.collection_id = 1060
+  and
+      awards_summary.award_value_amount > 0 --filter out awards with no value
+  and
+      awards_summary.award_status = 'active'
+  group by
+      buyer_identifier,
+      buyer_name,
+      award_value_currency
+  order by
+      award_amount desc
+  limit
+      10;
+
+Querying other collections and fields
+-------------------------------------
+
+Coverage of the OCDS schema varies by publisher.
```

Comment:
Can we take this for granted? It interrupts the flow of the how-to guide to lead with this fact.

Reply:
I don't think we can take it for granted, a common misconception amongst implementers is that they have to fill all the fields in the standard, so users may think the same.

I've moved it in https://github.com/open-contracting/kingfisher-views/pull/109/commits/ed554bbeca1617577f59d2ee115051cd6c4bb8c1 to improve the flow.

Reply:
I think the publisher and user perspectives on this point are different. Most users' first question is what the scope of a dataset is.